internal/clientcache/internal/cache/repository_targets.go

```diff
@@ -118,44 +129,56 @@ func (r *Repository) refreshTargets(ctx context.Context, u *user, tokens map[Aut
 	}
 
 	var numDeleted int
-	_, err = r.rw.DoTx(ctx, db.StdRetryCnt, db.ExpBackoff{}, func(_ db.Reader, w db.Writer) error {
-		var err error
-		switch {
-		case oldRefreshToken == nil || unsupportedCacheRequest:
-			if numDeleted, err = w.Exec(ctx, "delete from target where fk_user_id = @fk_user_id",
-				[]any{sql.Named("fk_user_id", u.Id)}); err != nil {
-				return err
-			}
-		case len(removedIds) > 0:
-			if numDeleted, err = w.Exec(ctx, "delete from target where id in @ids",
-				[]any{sql.Named("ids", removedIds)}); err != nil {
-				return err
-			}
-		}
-		switch {
-		case unsupportedCacheRequest:
-			if err := upsertRefreshToken(ctx, w, u, resourceType, sentinelNoRefreshToken); err != nil {
-				return err
+	var numUpserted int
```

Comment:
I'm not seeing this set anywhere, but it's later used to write a sys event

Reply:
Ah thanks, fixed